packages/docs/components/text-input.md

```diff
@@ -173,6 +173,67 @@ Note, when indicating a validation error, please use a `.ods-field--error` to in
 
 ## Additional types
 
+### Standalone Search
+
+> `<input>` elements of type search are text fields designed for the user to enter search queries into. These are functionally identical to text inputs, but may be styled differently by the user agent. - <cite><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search">MDN</a></cite>
+
+Odyssey's standalone search is styled to provide low-impact UI while maintaining accessibility when searching outside of normal form contexts. Inputs with `type="search"` will render with the "Search" UI indicator as well as a visually hidden label.
```

Comment:
> low-impact UI

Not sure what this means, would suggest removing or rephrasing. Maybe "minimal" here?

Reply:
> with the "Search" **UI indicator** as well

We should link to the "Icon vs UI Indicator" section of our icon docs.

Reply:
Good call; this is pretty rough. @joejezowski-okta @johnintrater-okta  any thoughts on phrasing here?